### ISSUE TITLE
Potential issue about dataloader

### ISSUE THREAD

**Issue by csyanbin** [crashmoon/MAML-Pytorch-Multi-GPUs#2]
For dataloader, number of workers is set to 4. Since the random seed is fixed, each worker has the same random state, and thus generate same classes for all 4 meta tasks. 
For example,
[49 54 32 36 40]
[49 54 32 36 40]
[49 54 32 36 40]
[49 54 32 36 40]
in the first iteration.
